src/ast/CMakeLists.txt

```diff
@@ -36,16 +36,19 @@ if (STATIC_LINKING)
       clangEdit
       clangFormat
       clangFrontend
+      clangIncludeFixerPlugin
       clangIndex
       clangLex
       clangParse
       clangRewrite
       clangSema
       clangSerialization
+      clangTidyPlugin
       clangToolingCore)
 
   set(llvm_lib_names
       bpfcodegen
+      frontendopenmp
```

Comment:
Nothing in particular, they're just now a requirement on the newer alpine version for the static build to work. There are references from clang to these libs that would be left unresolved otherwise.

I really want to find a better way to express the static llvm build, all these details leaking to us is not great.